### PR TITLE
static-info enhancement

### PR DESCRIPTION
**Pull request by V-Dandekar**
- Remove deprecated GetObjStaticInfo
- Update GetStaticInfo to include include-child-named-objects and
list-size for fixed sized lists
- Add mixins to create named objects based on user_creatable attribute.

https://tfs.ansys.com:8443/tfs/ANSYS_Development/Portfolio/_workitems/edit/682052

### CODE REVIEW

**Comment by V-Dandekar** [comment]
related fluent change
https://tfs.ansys.com:8443/tfs/ANSYS_Development/FluentDev/_git/root.git/pullrequest/331416

**Comment by mkundu1** [comment]
@V-Dandekar Is there a way to make this change so that it also works with Fluent 22.2?

#603 will enable the CI for Fluent 23.1. The CI will still fail as pyfluent depends on ansys-api-fluent==0.2.0 - the released version. We can make [https://github.com/ansys/ansys-api-fluent/pull/5](https://github.com/ansys/ansys-api-fluent/pull/5) into a dev release (ansys-api-fluent==0.2.dev0) and make pyfluent depend on that to pass the CI.

**Comment by V-Dandekar** [comment]
@mkundu1 CI has passed

**Review by mkundu1 — APPROVED** [review]

File: src/ansys/fluent/core/services/settings.py [hunk 1]
```diff
@@ -249,30 +229,37 @@ def _extract_static_info(self, info):
...
+        try:
+            if info.include_child_named_objects:
+                ret["include_child_named_objects"] = info.include_child_named_objects
+        except AttributeError:
+            pass
 
-        return self._extract_info(response.info)
+        try:
+            if info.list_size:
+                ret["list_size"] = info.list_size
```
Comment (h-krishnan) [inline src/ansys/fluent/core/services/settings.py:241]: @V-Dandekar Does ret["list_size"] get set to 0 here for settings which are not fixed size?

Comment (V-Dandekar) [reply src/ansys/fluent/core/services/settings.py:241]: no.

**Review by h-krishnan — APPROVED** [review]

**Comment by V-Dandekar** [comment]
Related ADO ticket https://tfs.ansys.com:8443/tfs/ANSYS_Development/Portfolio/_workitems/edit/682052

**Pull request merged**